File: problems/easy/tax_error/__tests__/tax_error.js
import {calculateTax} from "../tax_error"

const tests = [10000,0,-500];
const expected = [1500,0,0];
for (let index = 0; index < 3; index++) {
    if (calculateTax(tests[index]) != expected[index]) {
        throw new Error("❌ Test failed.");
    }
    
}
console.log("✅ JavaScript test passed!");

File: problems/easy/tax_error/__tests__/tax_error.py
from problems.easy.tax_error import calculate_tax

testcase = [10000,0,-500]
expected = [1500,0,0]
def test():
    passed = True
    for x in range(3):
        if calculate_tax(testcase[x]) != expected[x]:
            passed = False
            print("❌ python text failed")
            break
    
    print("✅ Python test passed!")

if __name__ =="__main__":
    test()